service/metrics/index_metrics.go

```diff
@@ -36,46 +36,45 @@ type MetricsWriter struct {
 
 // NewMetricsWriter creates a counter that counts indexed elements and exposes this information
 // as prometheus counters.
-func NewMetricsWriter(write *Writer) *MetricsWriter {
+func NewMetricsWriter(write *index.Writer) *MetricsWriter {
 	blockOpts := prometheus.CounterOpts{
```

Comment:
nit: consider adding Namestpace instead of a text prefix

Reply:
Done!